Dangerfile

```diff
@@ -19,3 +19,5 @@ no_changelog_entry = !git.modified_files.include?("CHANGELOG.md")
 if has_app_changes && no_changelog_entry && not_declared_trivial
   fail("Any changes to library code need a summary in the Changelog.")
 end
+
+jazzy.check fail: :modified
```

Comment:
@ilyapuchka sorry, I'm not sure I got the reason to not check `added` files for documentation.

> If we would want for any reason to keep something undocumented, other PRs should not be affected by that. And if everything is documented there will be no problem at all.

As I understood the point (but I'm not sure) - we could want to add new file to Sourcery and keep it undocumented so the danger will pass the PR. But I think once somebody decides to modify that file - danger will fail the PR. So As I've mentioned in the beginning of the comment, I'm confused 😕